Tests/js/views/services/assets/ez-contenteditviewservice-tests.js

```diff
@@ -405,8 +405,65 @@ YUI.add('ez-contenteditviewservice-tests', function (Y) {
         },
     });
 
+    getEditViewParametersTest = new Y.Test.Case({
+        name: "eZ Location View View Service getViewParameters tests",
+
+        setUp: function () {
+            this.app = new Y.Mock();
+            this.content = {};
+            this.contentType = {};
+            this.location = {};
+            this.owner = {};
+            this.version = {};
+            this.closeRedirectionUrl = {};
+            this.discardRedirectionUrl = {};
+            this.publishRedirectionUrl = {};
+            this.languageCode = {};
+            this.config = {};
+            this.service = new Y.eZ.ContentEditViewService({
+                content: this.content,
+                contentType: this.contentType,
+                location: this.location,
+                config: this.config,
+                owner: this.owner,
+                version: this.version,
+                closeRedirectionUrl: this.closeRedirectionUrl,
+                discardRedirectionUrl: this.discardRedirectionUrl,
+                publishRedirectionUrl: this.publishRedirectionUrl,
+                languageCode: this.languageCode,
+            });
+        },
+
+        tearDown: function () {
+            this.service.destroy();
+            delete this.service;
+            delete this.app;
+            delete this.content;
+            delete this.contentType;
+            delete this.location;
+            delete this.config;
+            delete this.owner;
+            delete this.version;
+            delete this.closeRedirectionUrl;
+            delete this.discardRedirectionUrl;
+            delete this.publishRedirectionUrl;
+            delete this.languageCode;
+        },
+
+        "Should get the view parameters": function () {
+            var params = this.service.getViewParameters();
+
+            Y.Assert.areSame(this.content, params.content, 'Service should get the content');
```

Comment:
The message should rather be something like: 'The content should be available in the return value of getViewParameters' (same for in the following lines)

In addition, there's a missing check on `mainLocation`.